lib/src/signed_video_h26x_sign.c

```diff
@@ -218,6 +218,8 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
 
   svrc_t status = SV_UNKNOWN_FAILURE;
   SV_TRY()
+    // Hash the |nalu_hash_list| before write the computed hash to TLV.
+    SV_THROW(hash_the_hash_list(self));
```

Comment:
This function is named `generate_sei_nalu()` and should only focus on generating the SEI data given the state in `self`. Hence, I suggest that this line should be moved outside.